feedback/utils.py

```diff
@@ -0,0 +1,8 @@
+"""Utilities for feedback app"""
+
+
+def _(text):
+    """
+    Make '_' a no-op, so we can scrape strings
```

Comment:
Following are better docs
https://github.com/openedx/xblock-drag-and-drop-v2/blob/db6130aeaa8e0b909b365f0fd290ec1183caccdb/drag_and_drop_v2/utils.py#L15